torchani/data/new.py

```diff
@@ -0,0 +1,444 @@
+import numpy as np
+import torch
+import functools
+from ._pyanitools import anidataloader
+import pkbar
+
+
+class CacheDataset(torch.utils.data.Dataset):
+
+    def __init__(self, file_path,
+                 species_order='HCNO',
+                 transform=True,
+                 self_energies=[-0.600953, -38.08316, -54.707756, -75.194466],
+                 batch_size=1000,
+                 device='cpu',
+                 bar=20,
```

Comment:
`bar` is confusing, it sounds like a progress bar. Why not calling it `chunking_threshold`?